tethysext/atcore/models/file_database/file_database_client.py

```diff
@@ -71,3 +77,78 @@ def path(self, root_dir: str) -> None:
         root_dir (str): the directory to be the root directory of the file database.
         """
         self._path = os.path.join(root_dir, str(self.instance.id))
+
+    def get_collection(self, collection_id: uuid.UUID) -> FileCollectionClient:
+        """
+        Get a FileCollectionClient owned by this FileDatabase by its collection_id
+
+        Args:
+            collection_id (uuid.UUID): The id for the file collection owned by this FileDatabase.
+
+        Returns:
+            The FileCollectionClient for the FileCollection.
+        """
+        collection_query = self._session.query(FileCollection).filter_by(
+            id=collection_id, file_database_id=self.instance.id
+        )
+        if collection_query.count() == 0:
+            raise FileCollectionNotFoundError(f'Collection with id "{str(collection_id)}" could not '
+                                              f'be found with this database.')
+        file_collection = collection_query.first()
+        collection_client = FileCollectionClient(
+            self._session, file_collection.id
+        )
+        return collection_client
+
+    def new_collection(self, items: list = None, meta: dict = None) -> FileCollectionClient:
+        """
+        Create a new collection copying any files and meta data passed in.
+
+        Args:
+            items (list): A list of files or paths to be copied to the file collection.
+            meta: (dict): The meta data to be stored in the FileCollection
+
+        Returns:
+            A new FileCollectionClient object for the FileCollection.
+        """
+        meta = meta or dict()
+        items = items or list()
+        new_collection = FileCollectionClient.new(self._session, self.instance.id, meta)
+        for item in items:
+            new_collection.add_item(item)
```

Comment:
We need to catch our own errors if one of the items is not found and rollback (delete the new FileCollection db entry and directory).